spec/core_functions/color/alpha.hrx

```diff
@@ -96,6 +102,16 @@ a {
   b: opacity(10%);
 }
 
+<===>
+================================================================================
+<===> opacity/with_css_var/input.scss
+a {b: opacity(var(--c))}
```

Comment:
Also test other special number cases (a calculation or an unquoted string that begins with `calc(`)